web/themes/joinup/templates/layout/page.html.twig

```diff
@@ -62,53 +62,57 @@
 
   {%
   set main_classes = [
-    'main-container',
-    'mdl-grid',
-    not page.sidebar_left ? 'main-container--center',
+    not page.sidebar_left ? 'mdl-grid--center',
   ]
   %}
-  <div class="section section--content">
-    <main class="{{ main_classes|join(' ') }}" role="main">
+  <main class="section section--content" role="main">
+    <div class="main-container">
       <a id="main-content" tabindex="-1"></a>{# link is in html.html.twig #}
+      <div class="row">
+        <div class="mdl-grid {{ main_classes|join(' ') }}">
+          {% if page.sidebar_left and not has_vertical_tabs %}
+            <aside class="section section--sidebar-left mdl-cell mdl-cell--2-col" role="complementary">
+              {{ page.sidebar_left }}
+            </aside>
+          {% endif %}
 
-      {% if page.sidebar_left and not has_vertical_tabs %}
-        <aside class="section section--sidebar-left mdl-cell mdl-cell--2-col" role="complementary">
-          {{ page.sidebar_left }}
-        </aside>
-      {% endif %}
+          <div class="section section--main mdl-cell {{ has_vertical_tabs ? 'mdl-cell--12-col mdl-grid--no-spacing content-container--form' : 'mdl-cell--8-col content-container' }}">
+            {% if page.content_top %}
+              <div class="section section--content-top {{ has_vertical_tabs ? 'mdl-cell mdl-cell--8-col mdl-cell--2-offset-desktop form__cell'}}">
```

Comment:
This part for content_top can be improved a bit:
- the check for region does not cover cases with empty content I think:
![empty_div](https://user-images.githubusercontent.com/1923476/28418767-34cd8006-6d5d-11e7-8885-014c49e81cce.png)
- as the `mdl-cell--2-offset-desktop` is the only option, please double test this. Maybe a similar [operator](https://twig.symfony.com/doc/2.x/templates.html#other-operators) will be better. Please double-test the offset will be handled correctly on absence.